src/app/components/AmpExperiment/index.tsx

```diff
@@ -18,8 +18,14 @@ type AmpExperimentConfig = {
   };
 };
 
+type AmpAnalyticsConfig = {
+  requests: Record<string, unknown>;
+  triggers: Record<string, unknown>;
```

Comment:
**Question (non-blocking)** Can this be typed any more strictly? I can see that we set base, clicks, trackClicks which we could encode here too.